<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>q_android_silence_audio_recording</class>
 <widget class="QMainWindow" name="q_android_silence_audio_recording">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>q_android_silence_audio_recording</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="q_frame_double_click" name="m_hl_top_menu">
      <property name="accessibleName">
       <string>top_frame</string>
      </property>
      <layout class="QHBoxLayout" name="l_hl_top_menu">
       <item>
        <widget class="QCheckBox" name="m_cb_minimized">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="m_hs_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_lb_title">
         <property name="text">
          <string>Android Silence Audio Recording</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="m_hs_0">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="m_cb_exit">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="m_hl_2">
      <item>
       <widget class="QWidget" name="m_w_info" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="accessibleName">
         <string>bottom_frame</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="m_lw_devices"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_pb_options">
           <property name="text">
            <string>Options</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>q_frame_double_click</class>
   <extends>QFrame</extends>
   <header>q_frame_double_click.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pb_options</sender>
   <signal>clicked()</signal>
   <receiver>q_android_silence_audio_recording</receiver>
   <slot>options()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>options()</slot>
 </slots>
</ui>
